tv_apps/dashboard/manifest.webapp

```diff
@@ -19,5 +19,10 @@
     "336": "/style/icons/dashboard_336.png",
     "672": "/style/icons/dashboard_672.png"
   },
-  "orientation": "landscape"
+  "orientation": "landscape",
+  "permissions": {
+    "browser": {},
+    "embed-widgets": {},
+    "webapps-manage": {}
```

Comment:
Got it.
Since I'm going to open a bug for embedding widget into dashboard, I'll include these codes there.